src/drawer/components/OverviewImage/OverviewImage.tsx

```diff
@@ -0,0 +1,78 @@
+import { ChangeEvent, useState } from 'react';
+
+import { IcXLine, IconContext } from '@yourssu/design-system-react';
+import { useTheme } from 'styled-components';
+
+import {
+  StyledDeleteFileBtn,
+  StyledFileUploadLabel,
+  StyledImageUpload,
+  StyledOverviewContainer,
+  StyledOverviewDescription,
+  StyledOverviewTitle,
+  StyledOverviewTitleContainer,
+  StyledOverviewUpload,
+} from './OverviewImage.style';
+
+export const OverviewImage = () => {
+  const [files, setFiles] = useState<File[]>([]);
+  const theme = useTheme();
+
+  const handleFileChange = (file: File | undefined, index: number) => {
+    setFiles((prevFiles) => {
+      const newFiles = [...prevFiles];
+      if (file !== undefined) {
+        newFiles[index] = file;
+      }
+      return newFiles;
+    });
```

Comment:
```suggestion
setFiles((prevFiles) => prevFiles.map((prevFile, i) => i === index ? file : prevFile));
```
이 부분도 `Array.prototype.map()` 사용해서 업데이트하는게 깔끔한 것 같습니다!

Reply:
오,,, 너무 좋은데요 짱짱입니다

Reply:
![image](https://github.com/yourssu/Soomsil-Web/assets/94633589/8860d3e5-9cc7-4a8d-895b-aef30586fc1a)